MdePkg/Library/UefiLib/UefiLibPrint.c

```diff
@@ -425,7 +425,13 @@ InternalPrintGraphic (
   }
 
   Blt = (EFI_IMAGE_OUTPUT *)AllocateZeroPool (sizeof (EFI_IMAGE_OUTPUT));
-  ASSERT (Blt != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (Blt == NULL) {

```

Comment:
Although this currently effectively accomplishes the same thing as `goto Error`, I recommend using the goto so it follows the common error exit path. If additional code is added there in the future, this will automatically go through it.